sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/GenericArrayData.scala

```diff
@@ -38,7 +40,9 @@ class GenericArrayData(val array: Array[Any]) extends ArrayData {
   def this(primitiveArray: Array[Boolean]) = this(primitiveArray.toSeq)
 
   def this(seqOrArray: Any) = this(seqOrArray match {
-    case seq: Seq[Any] => seq.toArray
+    // Specified this as`scala.collection.Seq` because seqOrArray can be
+    // `mutable.ArraySeq` in Scala 2.13
+    case seq: scala.collection.Seq[Any] => seq.toArray
```

Comment:
The entrance is

https://github.com/apache/spark/blob/04f7f6dac0b9177e11482cca4e7ebf7b7564e45f/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/ArrayData.scala#L28-L38

not easy to call `toSeq`, so I changed it here